crypto/fipsmodule/rand/urandom_test.cc

```diff
@@ -437,10 +438,10 @@ static std::vector<Event> TestFunctionPRNGModel(unsigned flags) {
       // non-blocking OS entropy draw will be tried.
       (!have_fast_rdrand() && !have_fork_detection() &&
        !sysrand(false, kAdditionalDataLength)) ||
-      // Opportuntistic entropy draw in FIPS mode because RDRAND was used.
-      // In non-FIPS mode it's just drawn from |CRYPTO_sysrand| in a blocking
-      // way.
-      !sysrand(!is_fips, CTR_DRBG_ENTROPY_LEN) ||
+      // Initialise CRNGT. In FIPS mode we use a blocking sysrand call for
+      // a personalization string, while in non-FIPS mode the same call is
+      // used for generating entropy for a seed.
+      !sysrand(true, kSeedLength) ||
```

Comment:
In [rand.c, L328-L337](https://github.com/awslabs/aws-lc/blob/6c9bc3a6b8cd8592ff903084e8a7ee54c0fda6ba/crypto/fipsmodule/rand/rand.c#L328-L337), it's mentioned that, in non-FIPS mode, there is a non-blocking sysrand call, `CRYPTO_sysrand_if_available()`,  to fill the personalization string. I believe this call doesn't currently occur since `used_cpu` is set to 0 by the prior call to the **non-fips** `rand_get_seed()`. If we want to fill the personalization string, we should remove `used_cpu` from the `if` condition. 
And if we do that, then another line should be added here after l.444
```
(!is_fips && !sysrand(false, CTR_DRBG_ENTROPY_LEN))
```
(I hope I understood the sequence correclty.)

Reply:
Makes sense. As discussed, thank you for removing in 20e965d the dead code linked above.